demisto_sdk/commands/create_artifacts/content_artifacts_creator.py

```diff
@@ -296,6 +305,26 @@ def add_widget_as_content_item(self, content_object: ContentObject):
             'widgetType': content_object.get('widgetType', '')
         })
 
+    def add_generic_field_as_content_item(self, content_object: ContentObject):
+        self.content_items[ContentItems.GENERIC_FIELD].append({
+            'name': content_object.get('name', '')
+        })
+
+    def add_generic_type_as_content_item(self, content_object: ContentObject):
+        self.content_items[ContentItems.GENERIC_TYPES].append({
+            'name': content_object.get('name', '')
+        })
+
+    def add_generic_definition_as_content_item(self, content_object: ContentObject):
+        self.content_items[ContentItems.GENERIC_DEFINITIONS].append({
+            'name': content_object.get('name', '')
+        })
+
+    def add_generic_module_as_content_item(self, content_object: ContentObject):
+        self.content_items[ContentItems.GENERIC_MODULES].append({
+            'name': content_object.get('name', '')
+        })
```

Comment:
is name enough though? Where are the other fields used?

Reply:
these are the fields in the metadata file of each pack. updated according to- https://github.com/demisto/etc/issues/38101#issuecomment-882368058

Reply:
Is that what is displayed in the marketplace? if so let's also add the description there - @Itay4 anything else you can think of?

Reply:
Right, its what is displayed in the marketplace. description is there- https://github.com/demisto/demisto-sdk/blob/07f5c9b1340adb995ac439e12a406ca74f7af57d/demisto_sdk/commands/create_artifacts/content_artifacts_creator.py#L323-L332